### PR TITLE
Prototyping in Figma

### PR DESCRIPTION
**Pull request by dalshekerchi**
This document provides valuable tips and additional information for effective prototyping in Figma. This enhancement offers users practical advice, best practices, and insights into advanced features and collaboration techniques within Figma.

### CODE REVIEW

File: Topics/User_Experience/Prototyping.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Guide to Using Prototyping in Figma
+
+## Introduction to Figma
+
+### What is Figma?
+Figma is a cloud-based design tool that revolutionizes the way teams create, collaborate, and share work on user interface and user experience projects. It combines powerful design capabilities with a user-friendly interface, making it accessible for both experienced designers and those new to design.
+
+### Key Features of Figma
```
Comment (hellokathylee) [inline Topics/User_Experience/Prototyping.md:8]: maybe here explain why one would want to prototype their work on Figma

File: Topics/User_Experience/Prototyping.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### Step 8: Sharing for Collaboration
+- **Share for Feedback**: Easily share prototypes with team members and stakeholders for valuable feedback.
+- **Collaborative Review Sessions**: Conduct live review sessions where team members can provide input in real-time.
+
+## Conclusion
+
+Prototyping in Figma is a critical step in the design process. It allows for testing, iteration, and communication, bridging the gap between design and development. By mastering Figma’s prototyping capabilities, designers can create more effective and user-friendly designs.
+
+
```
Comment (hellokathylee) [inline Topics/User_Experience/Prototyping.md:69]: if you have other topics to cover, include an additional resources/tips & tricks section

File: Topics/User_Experience/Prototyping.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- **Unified Vision**: Figma helps teams align on a shared vision of the product, ensuring everyone understands the goals and design direction.
+- **Accessibility for Non-Technical Team Members**: Its intuitive interface allows individuals without a technical background to participate actively in the design process.
+- **Real-Time Updates**: Team members can see changes as they happen, ensuring everyone is always looking at the latest version of the design.
+
+### Commenting and Feedback in Figma
+- **Interactive Feedback**: Team members can leave comments directly on the design, making it easy to track feedback and revisions.
+- **Iterative Design Process**: Comments allow for an iterative approach, where designs are refined based on continuous team input.
+- **Inclusive Design Reviews**: Stakeholders can contribute feedback, making the design process more democratic and inclusive.
+
+## In-Depth Guide to Prototyping in Figma
```
Comment (hellokathylee) [inline Topics/User_Experience/Prototyping.md:25]: add some images for better understanding?

File: Topics/User_Experience/Prototyping.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- **Linking Frames**: Connect frames to depict user flow and interactions.
+- **Customizing Transitions**: Select from various animation options for smooth transitions.
+
+### Step 6: Advanced Prototyping Techniques
+- **Overlay**: Create modals, menus, or dialogs that overlay other frames.
+- **Scrolling**: Implement scrolling to simulate long or wide content.
+- **Device Frames**: Test your designs in specific device frames for realism.
+
+### Step 7: Testing and Iteration
+- **Preview Mode**: Use the 'Present' mode to interact with your prototype.
```
Comment (hellokathylee) [inline Topics/User_Experience/Prototyping.md:57]: if adding pictures doesn't work out, maybe describe the icons or something so the user can make out what to look out for

**Review by hellokathylee — CHANGES_REQUESTED** [review]
also add to README!

**Review by dalshekerchi — COMMENTED** [review]
Thanks for the feedback! I implemented all the comments that you pointed out, including adding a couple of new sections as well as adding an image. Let me know if there is anything else that needs to be done :)

**Review by hellokathylee — APPROVED** [review]
Looks good to me! Nicely done!!

**Pull request merged**